tests/integration/test_arctic_multithreading.py

```diff
@@ -6,14 +6,20 @@
 
 from arctic import Arctic, VERSION_STORE
 from arctic.exceptions import LibraryNotFoundException
+from arctic.hooks import register_get_auth_hook
 from arctic.store.version_store import VersionStore
 
 MY_ARCTIC = None  # module-level Arctic singleton
+AUTH_COUNT = 0
 
-
```

Comment:
PEP-8 gets a bit sad if you don't have two blank lines before top-level functions

Reply:
fixed